invenio_accounts/profiles/schemas.py

```diff
@@ -21,6 +23,25 @@ def validate_visibility(value):
         )
 
 
+def validate_locale(value):
+    """Check if the value is a valid locale."""
+    locales = current_app.extensions["invenio-i18n"].get_locales()
+    locales = [locale.language for locale in locales]
```

Comment:
I tried this in one env, and I get:
```
In [2]: locales = current_app.extensions["invenio-i18n"].get_locales()
In [3]: locales
Out[3]: [Locale('en')]
In [4]: locales = [locale.language for locale in locales]
In [5]: locales
Out[5]: ['en']
```
I don't think `locale.language` is correct. I would use a `set()` and not a `list`.
I would not add the default `locales.append("en_GB")`: you might have an instance that has only one configured locale, e.g. Italian :)

Reply:
I would:
1. understand if `flask-babel` is doing something particular with the variable `BABEL_DEFAULT_LOCALE`, e.g. setting some global context in flask
2. eventually, change [the func](https://github.com/inveniosoftware/invenio-i18n/blob/master/invenio_i18n/ext.py#L170) in `invenio-i18n` (pseudocode):

```diff
- langs = [self.babel.default_locale]
- for lang, dummy_title in current_app.config.get("I18N_LANGUAGES", []):
    - langs.append(Locale.parse(lang))
+ # is the python obj Locale implpementing the magic methods to compare, to avoid to add duplicates in a `set`?
+ locales = set()
+ # the following should be a locale, not a language only (`en_GB`, not `en`)
+ default_locale = current_app.config.get("BABEL_DEFAULT_LOCALE", self.babel.default_locale)
+ for ... loop
```

The question is: if I set language `en`, what will be the locale? `en_GB` or `en_US`, for example?